src/pac_case.cc

```diff
@@ -351,10 +346,16 @@ void CaseField::GenCleanUpCode(Output* out_cc, Env* env) {
     GenCaseStr(index_, out_cc, env, case_type()->IndexExpr()->DataType(env));
     out_cc->inc_indent();
     out_cc->println("// Clean up \"%s\"", id_->Name());
-    out_cc->println("{");
-    if ( ! anonymous_field() )
+    if ( ! anonymous_field() ) {
```

Comment:
No biggie but this can apparently still leave empty pairs of curly braces, like this example from Zeek's `websocket_pac.cc`:
```
WebSocket_FrameHeader::~WebSocket_FrameHeader() {
    switch ( maybe_more_len_case_index() ) {
        case ((uint8)126):
            // Clean up "payload_len2"
            {
            }
            break;
        case ((uint8)127):
            // Clean up "payload_len8"
            {
            }
            break;
        default:
            // Clean up "short_len"
            {
            }
            break;
    }
```
Just in case you want to iterate on that one.

Reply:
We shouldn't even generate those `case` blocks if they're not doing anything. Unfortunately we can't know they're not going to be blank until we call `GenCleanUpCode` on the object, which does it's own output to the file. There's a lot of places in binpac where that kind of thing happens.